### PR TITLE
i18n for helms

### PR DESCRIPTION
**Pull request by aBlueShadow**
Prepare translation for helms screen.

### CODE REVIEW

File: src/screens/crew6/helmsScreen.cpp [hunk 1]
```diff
@@ -126,7 +127,7 @@ void HelmsScreen::onDraw(sf::RenderTarget& window)
         energy_display->setValue(string(int(my_spaceship->energy_level)));
         heading_display->setValue(string(my_spaceship->getHeading(), 1));
         float velocity = sf::length(my_spaceship->getVelocity()) / 1000 * 60;
-        velocity_display->setValue(string(velocity, 1) + DISTANCE_UNIT_1K + "/min");
+        velocity_display->setValue(string(velocity, 1) + DISTANCE_UNIT_1K + tr("unit", "/min"));
```
Comment (daid) [inline src/screens/crew6/helmsScreen.cpp]:
```
string(velocity, 1) + DISTANCE_UNIT_1K + tr("unit", "/min")
```
Would be better as:
```
tr("{value} {unit}/min").format({{"value", string(velocity, 1)}, {"unit": DISTANCE_UNIT_1K}})
```
I think. As that would add flexibility to the translator, and a bit more context where this is used.

File: src/screenComponents/jumpControls.cpp [hunk 2]
```diff
@@ -18,10 +19,10 @@ GuiJumpControls::GuiJumpControls(GuiContainer* owner, string id)
     charge_bar->setPosition(0, -50, ABottomLeft)->setSize(50, GuiElement::GuiSizeMax);
     charge_bar->hide();
     
-    label = new GuiKeyValueDisplay(this, id + "_LABEL", 0.5, "Distance", "10.0");
+    label = new GuiKeyValueDisplay(this, id + "_LABEL", 0.5, tr("Distance"), "10.0");
```
Comment (daid) [inline src/screenComponents/jumpControls.cpp]: Might be good to have a context with this distance? Not all distances might translate the same in all languages?

Comment (aBlueShadow) [reply src/screenComponents/jumpControls.cpp]: You are right. I actually added the context in another part of the file, but missed that one.

File: src/screens/crew6/helmsScreen.cpp (hunk 1 above)
Comment (aBlueShadow) [reply src/screens/crew6/helmsScreen.cpp]: Done. (I guess the **:** was a typo)

File: src/screenComponents/jumpControls.cpp (hunk 2 above)
Comment (aBlueShadow) [reply src/screenComponents/jumpControls.cpp]: Done.

**Pull request merged**